docs/src/pages/guides/aliases.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Aliases
+---
+
+Aliasing is an integral part when it comes to clean and readable code, that's why Astro comes with support for custom aliases out of the box.
```

Comment:
```suggestion
Aliasing is an integral part when it comes to clean and readable code. Astro comes with support for custom aliases out of the box. Allowing you to maintain a clean and organised project. 
```

Reply:
Actually, I'd remove the first line entirely as too opinionated :) Some people think that aliases are Bad™️, and our project doesn't really take a stance one way or the other.

Instead, maybe use the first sentence to explain what an alias is and why you might want to use it. 

Love that you make the point early that Astro supports aliases "out of the box".